src/examples/ParkingLot/Input/ParkingLot.xml

```diff
@@ -0,0 +1,5130 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
In the measurements, they don't see Ground so in the final Parking Lot scenario, I removed the ground plane now. We don't have QD parameters for Ground in the parking lot. If you still want, I can include Ground that covers the entire buildings. Please let me know.

Reply:
I know we don't have the ground parameters, but please try to ask Camillo about a sensible value (or range) for the reflection loss. We can set the rest of the QD parameters to 0 so that no pre/post cursors will appear but ground reflection losses should really be taken into account.
So no, I would not remove the ground plane but actually extend it to cover the whole building area.

Reply:
For the current Rx locations, we see ground in the current XML file. In future, I will change this.

Reply:
these scenarios could be used by a user to do their own simulations with custom mobility. changing it now requires almost zero effort, so I would encourage you to do it now rather than at a later time

Reply:
Okay! Dont worry I will do it now.

Reply:
Done